### PR TITLE
python: improvements to config handling

### PR DESCRIPTION
**Pull request by mlyle**
dronin-getconfig now outputs an XML file, new dronin-putconfig can store one to the FC, improvements to the telemetry class's handling of nacks and saving.  Infrastructure so that client programs can extend the auto-parsing without providing an entire telemetry argument parser.

### CODE REVIEW

**Comment by mlyle** [comment]
jenkins, test this please

**Comment by mlyle** [comment]
jenkins, test this please

**Comment by mlyle** [comment]
jenkins, test this please

File: python/dronin-getconfig [hunk 1]
```diff
@@ -57,15 +57,18 @@ def main():
...
         if val is not None:
-            print(val)
+            sobjs.append(val)
         else:
             missing.append(s._name)
 
-    for s in missing:
-        print("No instance of %s" % (s))
+    #for s in missing:
+    #    print("No instance of %s" % (s))
```
Comment (tracernz) [inline python/dronin-getconfig:69]: We should use python's excellent logging class to do this stuff, then it's easy to switch on debug type stuff when needed. (to clarify, not asking you to go do that in this PR, just a suggestion for stuff we add, and stuff we clean up)

**Review by tracernz — APPROVED** [review]
Superb. :+1:

File: python/dronin-getconfig (hunk 1 above)
Comment (mlyle) [reply python/dronin-getconfig:69]: Yah, I've got that on my todo list for a big cleanup... but I'm trying to sprint towards integration tests and I kinda hate the python logging stuff (it definitely takes thought on how to instantiate it right in a framework like this etc)

**Pull request merged**